1223-curator_worklist/357-curator_opens_my_worklist.feature

```diff
@@ -56,4 +56,21 @@ Feature: Curator opens My Worklist
             | Registration title |
             | Submitter name     |
             | Date               |
-        And the see that each Worklist Item has an Expand button
\ No newline at end of file
+        And the see that each Worklist Item has an Expand button
+
+    @1252
+    Scenario: Curator views details of a Worklist item
+        Given that a Curator views DOI request tab
+        When they expand the item
+        Then they see the item is expanded
+        And they see the Title of the Publication
```

Comment:
```suggestion
        When they click on an item
        Then they see the Title of the Registration
```
tror dette vil gjøre selve testingen litt enklere, men beskrivelsen like god (@Eiriknil ?)

Reply:
OK, for min del